### ISSUE TITLE
Staging Review finding: Focus lost after file "Change file" upload

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#125957]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Form Upload
**Product:** Form Upload
**Feature:** Multi-file upload (Adding supporting evidence)

## Findings details
**VA.gov Experience Standard - issue:** [User experiences an unexpected interaction with a component or pattern.](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

When using keyboard alone, and selecting to "Change file" when the upload completes, focus is reset to the top of the page. This can be disorienting and confusing for users, and forces them to renavigate the page to get back to the task. (Desktop)

Note: With Chrome/Talkback, focus is reset to the top of the page on successful upload or on "Change file"

<details><summary>Link, screenshot or steps to recreate</summary>

https://github.com/user-attachments/assets/af86aed9-2e6d-4dfd-820b-f136c591adc4

</details>

## Recommended action

Ensure that focus is managed appropriately, and goes to a logical location when the user interacts with "Change file". With the initial upload, if the user clicks [TAB] after a successful upload, focus moves to the "Change file" button. 

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** [WCAG 2.4.3 Focus Order - Level A](https://www.w3.org/TR/WCAG22/#focus-order)
- **Modality:** N/A
- **Design System Component:** [Form - File input component](https://design.va.gov/components/form/file-input)
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by tbaker1026** [comment]
Will be resolved in - https://github.com/department-of-veterans-affairs/va.gov-team/issues/123070
